android/src/main/java/org/ligi/gobandroid_hd/logic/GoGame.kt

```diff
@@ -302,6 +302,47 @@ class GoGame @JvmOverloads constructor(size: Int, handicap: Int = 0) {
             }
         }
 
+    /**
+     * @return the last variations move
+     */
+    fun lastVarMove(): GoMove?
```

Comment:
no need to be sorry - was just going through the PR and this was one of the things that I thought should be changed before merging